docs/hardware/devices/seeed-studio/sensecap/card-tracker.mdx

```diff
@@ -34,8 +34,9 @@ SenseCAP T1000-E is a high-performance tracker designed for Meshtastic. As small
 - Purchase Links:
   - International
     - [Seeed Studio](https://www.seeedstudio.com/SenseCAP-Card-Tracker-T1000-E-for-Meshtastic-p-5913.html)
+- [User Guide](https://wiki.seeedstudio.com/sensecap_t1000_e/)
 
 #### Images
 
-![SenseCAP Card Tracker T1000-E for Meshtastic](/img/hardware/seeed/card_tracker.webp)
-![SenseCAP Card Tracker T1000-E Dimensions](/img/hardware/seeed/T1000-E.webp)
+<img src="http://files.seeedstudio.com/wiki/SenseCAP/Meshtastic/t1000-e-meshtastic.png" alt="SenseCAP Card Tracker T1000-E for Meshtastic" width="800" />
+<img src="http://files.seeedstudio.com/wiki/SenseCAP/Meshtastic/t1000-e.png" alt="SenseCAP Card Tracker T1000-E Dimensions" width="800" />
```

Comment:
Please include images on the local repository instead of linking them.

Reply:
updated. thx